plugins/kubernetes/app/models/concerns/kubernetes/api/deleted_pod.rb

```diff
@@ -0,0 +1,9 @@
+module Kubernetes
+  module Api
+    module DeletedPod
```

Comment:
Some documentation for this class would be useful. It's not obvious what it's for at first glance.